src/vs/workbench/services/hover/browser/hoverService.ts

```diff
@@ -44,9 +44,10 @@ export class HoverService implements IHoverService {
 		}
 		this._currentHoverOptions = options;
 		this._lastHoverOptions = options;
-		options.trapFocus = options.trapFocus || this._accessibilityService.isScreenReaderOptimized();
+		// HACK, remove when #189076 is fixed
```

Comment:
I think we'll always want this `isScreenReaderOptimized` change, I mean for the hack to talk about the `skipLastFocusedUpdate` prop specifically (instead using the lock request approach)

Reply:
Oh I was confused because of the line you commented on. Makes sense 👍🏻

Reply:
I am also curious abt why for non screen reader users that list item in the explorer doesn't get focused on hover close. Isn't that desirable for everyone?

Reply:
I'm a little scared about the change as it is 😅, setting it true by default is much more risky. I think the reason it's opt-in is either because it was just added and didn't consider all cases, or you may not want it in some cases like when the hover shows on focus